data/docs/rules-v2.md

```diff
@@ -0,0 +1,204 @@
+title: Comment jouer ?
+
+Dans le Corporate Game, vous incarnez une pointure dans l’univers de la finance dans un futur où le capitalisme a fini par dissoudre les pays. Vous jouez à Detroit, la première ville libérée du contrôle des États. Bienvenue dans le Disneyland des traders !
```

Comment:
Si je ne m'abuse, on ne joue pas nécessairement des pointures dans l'univers de la finance. Et encore moins des traders. ^^"

On joue des execs, des cadres ambitieux aux profils divers, qui sont bien plus intéressé par le pouvoir que par le pognon.  "Not in the money business, in the empire business".